ios/FluentUI/Popup Menu/PopupMenuProtocols.swift

```diff
@@ -38,7 +38,7 @@ public protocol PopupMenuTemplateItem: AnyObject {
  You can use `UITableViewCell` conforms to this protocol for customization.
 */
 @objc(MSFPopupMenuItemTemplateCell)
-public protocol PopupMenuItemTemplateCell where Self: UITableViewCell {
```

Comment:
Did we ever figure out why this in particular wasn't valid? It still doesn't quite make sense why this isn't allowed. My best guess is that it doesn't work with the `@objc` attribute, but I haven't found explicit documentation for that limitation anywhere.

Reply:
Because it emits this code in the swift interface which is invalid. Probably worth a bug on Apple?
```
@objc(MSFPopupMenuItemTemplateCell) public protocol PopupMenuItemTemplateCell : UITableViewCell {
```

Reply:
But I guess, I still don't understand why that's invalid. Those are two different ways of expressing essentially the same thing in Swift.